### ISSUE TITLE
Fish expansion includes comment

### ISSUE THREAD

**Issue by ryanorendorff** [warpdotdev/Warp#3060]
### Describe the bug

When a fish expansion is defined with a comment in it:

```
$ abbr
...
abbr -a -- ga 'git add' # imported from a universal variable, see `help abbr`
...
```

if the expansion is used on the command line, the comment is also added.

```
# ga expands to the following
$ git add # imported from a universal variable, see `help abbr` 
```

The comment should not be included in the abbreviation expansion.

### To Reproduce

1. Define a fish abbreviation that includes help text.
2. Use the abbreviation.

The comment is included in the expansion.

### Expected behavior

The comment should not be included in the expansion.

### Operating System

MacOS

### Operating System and Version

macOS 13.3.1 (a)

### Shell Version

fish, version 3.6.1

### Warp Version

v0.2023.05.09.08.03.stable_01

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Thanks for letting us know @ryanorendorff. Sorry if this is a blocker for you, there doesn't seem to be an immediate workaround other than to not include help text in abbreviations you add, but I'll bring this up with the team and we'll post any updates on this thread.

Please add a :+1: to the original post at the top or comment with your details to signal that you're also having the same issue, and then hit the subscribe button if you'd like to be notified.

**Comment by ryanorendorff** [comment]
Ya no problem! I using warp and wezterm together since I have a lot of generated aliases that come from elsewhere, so I don't have direct access to edit the abbreviations.

**Comment by mwmcode** [comment]
[From fish docs](https://fishshell.com/docs/current/cmds/abbr.html)

"""
In that case you should take the part before the # comment and save it in [config.fish](https://fishshell.com/docs/current/language.html#configuration), then you can run abbr --erase to remove the universal variable:


```sh
abbr >> ~/.config/fish/config.fish
abbr --erase (abbr --list)
```
[restart]
"""
_If the issue persisted, open fish config and remove tail comments from the abbreviations there, save, close, restart 🤞_

**Comment by danielpeng2** [comment]
Thanks for your patience, a fix is coming out in this week's release to avoid including the comment when importing an alias from a universal variable.

**Comment by danielpeng2** [comment]
This should be fixed as of the latest release (`v0.2023.08.01.08.05.stable_03`). Closing this issue, but feel free to leave a comment if there's any more problems with this.
